frontend/src/app/modules/router/openproject.routes.ts

```diff
@@ -133,6 +135,15 @@ export function initializeUiRouterListeners(injector:Injector) {
         return $state.target(transition.to(), paramsCopy);
       }
 
+      // Set backRoute to know where we came from
+      if (fromState.name &&
+          fromState.data &&
+          toState.data &&
+          fromState.data.parent !== toState.data.parent) {
+          const paramsFromCopy = Object.assign({}, transition.params('from'));
```

Comment:
For a shallow copy, prefer the spread operator: `const paramsFromCopy = { ...transition.params('from') } since it's typesafe. (Even though It does not matter much in this line)